tests/e2e/e2e_permissive_trafficsplit_test.go

```diff
@@ -0,0 +1,299 @@
+package e2e
```

Comment:
I was wondering if it made sense to extend the existing e2e_trafficsplit_test.go that adds an additional variable in the top-level context for SMI vs permissive mode: https://github.com/openservicemesh/osm/blob/main/tests/e2e/e2e_trafficsplit_test.go#L24-L32. That way, we would only need to set the mode and everything in the test would remain the same, which makes it easier to maintain the test.

Reply:
yea I agree. That's a great idea.

Reply:
I extended the existing traffic split test to permissive mode in https://github.com/openservicemesh/osm/pull/4074/commits